recipe/meta.yaml

```diff
@@ -8,11 +8,10 @@ source:
   url:  http://sourceforge.net/projects/pyqt/files/PyQt5/PyQt-5.9.2/PyQt5_gpl-5.9.2.tar.gz
   sha1: 058c5b37981b515cc4e16faa04a4a6d8892cdaa9
   patches:
-    - 0001-Ignore-lack-of-shared-python-lib.patch
-    - 0002-configure.patch
+    - 0003-configure.patch
```

Comment:
Please add a comment above this line explaining the rationale of this patch.

Reply:
This patch instead of asking python for the location of its shared library goes with the assumption that it is there (as it is with Anaconda).
I removed the simple check and that is it.